src/modules/Dashboard/components/TakeNowModal/TakeNowModal.tsx

```diff
@@ -126,6 +137,11 @@ export const useTakeNowModal = ({ dataTestId }: UseTakeNowModalProps) => {
     [participantsOnly, teamMembersOnly],
   );
 
+  const fullAccountParticipantsAndTeamMembers = useMemo(
+    () => [...teamMembersOnly, ...fullAccountParticipantsOnly],
+    [fullAccountParticipantsOnly, teamMembersOnly],
+  );
```

Comment:
Related to the earlier comments — I think we should by filtering out duplicates that may appear in both `teamMembersOnly` and `fullAccountParticipantsOnly` arrays, here.

Reply:
Apparently I forgot to filter out team members from the `fullAccountParticipantsOnly` array, so I made that update.

cc @farmerpaul